File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.java
class Solution {
    public int subtractProductAndSum(int n){
        int sum =0;
        int pro =1;
        while(n>0){
            sum=sum+(n%10);
            pro=pro*(n%10);
            n=n/10;
        }
        return pro-sum;
    }
}

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.js
/**
 * @param {number} n
 * @return {number}
 */
var subtractProductAndSum = function(n) {
    let pro = 1;
    let sum = 0;
    while(n!=0){
        const rem = n%10;
        sum+=rem;
        pro*=rem;
        n=Math.floor(n/10);
    }
    return pro-sum;
};